target/linux/ath79/image/generic.mk

```diff
@@ -88,6 +88,15 @@ define Device/seama
   SEAMA_SIGNATURE :=
 endef
 
+define Device/telco_electronics_tel-t1
+  ATH_SOC := qca9531
+  DEVICE_TITLE := Telco Electronics T1
+  DEVICE_PACKAGES := kmod-leds-gpio kmod-usb2 kmod-usb-net \
+	kmod-usb-net-qmi-wwan -swconfig
+  IMAGE_SIZE := 16192k
+  SUPPORTED_DEVICES += t1
```

Comment:
This is only required if the device has been present in ar71xx. If not, remove the line entirely.

Reply:
Probably this can be set to a compatible name of the openwrt based vendor fw???